gtk3/sample/misc/textbuffer_serialize.rb

```diff
@@ -2,21 +2,18 @@
 =begin
   textbuffer_serialize.rb - Ruby/GTK sample script.
 
-  Copyright (c) 2006 Ruby-GNOME2 Project Team 
+  Copyright (c) 2005-2015 Ruby-GNOME2 Project Team
```

Comment:
Oups sorry. You can reject this merge request . I will make the modifications after that